content/docs/02-foundations/02-providers-and-models.mdx

```diff
@@ -81,6 +81,10 @@ Here are the capabilities of popular models:
 
 | Provider                                                                 | Model                                       | Image Input         | Object Generation   | Tool Usage          | Tool Streaming      |
 | ------------------------------------------------------------------------ | ------------------------------------------- | ------------------- | ------------------- | ------------------- | ------------------- |
+| [xAI Grok](/providers/ai-sdk-providers/xai)                              | `grok-3-beta`                               | <Cross size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
```

Comment:
are we sure all these cross + checks are correct? not just copied over I heard multimodal in other PR

Reply:
i checked these with the e2e tests -- if i said multimodal in the other PR we should correct it, AFAIK none of these accept image (or other) input yet, nor generate multimedia output yet.

Reply:
👍